src/Hooks/Hooks_global.h

```diff
@@ -0,0 +1,12 @@
+#ifndef HOOKS_GLOBAL_H
```

Comment:
Remove this file, it is not needed because your dll doesn't export anything

Reply:
Removed